pandas/io/common.py

```diff
@@ -857,12 +857,15 @@ def file_exists(filepath_or_buffer: FilePathOrBuffer) -> bool:
 
 def _is_binary_mode(handle: FilePathOrBuffer, mode: str) -> bool:
     """Whether the handle is opened in binary mode"""
+    # specified by user
+    if "t" in mode or "b" in mode:
```

Comment:
why is t here? is this tested?

Reply:
"t" is used to explicitly request text-mode but it usually omitted. One of `read_json` or `to_json` is using it. I will add a test with a handle that needs that.

Reply:
kk